bucket/rapidee.json

```
@@ -16,6 +16,8 @@
     },
     "homepage": "https://www.rapidee.com/en/about",
     "bin": "rapidee.exe",
+    "persist": "rapidee.ini",
+    "pre_install": "if(!(Test-Path \"$persist_dir\\rapidee.ini\")) {Set-Content -Encoding ASCII -Path \"$dir\\rapidee.ini\" -Value ''}",
```

No need to open that file and insert empty string. Just create it to save opening and saving.

```suggestion
    "pre_install": "if(!(Test-Path \"$persist_dir\\rapidee.ini\")) { New-Item \"$dir\\rapidee.ini\" -Type File }",
```

---

It works. It's strange that I tried `New-Item` and rapidee didn't recognize the generated empty file. 😂